application/src/main/java/org/togetherjava/tjbot/features/help/HelpSystemHelper.java

```diff
@@ -131,6 +143,92 @@ private RestAction<Message> sendExplanationMessage(GuildMessageChannel threadCha
         return action.setEmbeds(embeds);
     }
 
+    /**
+     * Gather first message from thread and pass into asynchronous sendChatGptAttempt.
+     * 
+     * @param threadChannel - The forum thread channel where the question originated
+     * @return A response from the AI service indicating success or failure.
+     * @see HelpSystemHelper#sendChatGptAttempt
+     */
+    RestAction<Message> prepareChatGptAttempt(ThreadChannel threadChannel) {
+        return threadChannel.retrieveMessageById(threadChannel.getIdLong())
+            .flatMap(message -> sendChatGptAttempt(message.getContentRaw(), threadChannel));
+    }
+
+    /**
+     * Determine between the title of the thread and the first message which to send to the AI. It
+     * uses a simple heuristic of length to determine if enough context exists in a question. If the
+     * title is used, it must also include a question mark since the title is often used more as an
+     * indicator of topic versus a question.
+     * 
+     * @param questionFirstMessage - The first message of the thread which originates from the
+     *        question asker.
+     * @param threadChannel - The thread in which the question was asked.
+     * @return An answer for the user from the AI service or a message indicating either an error or
+     *         why the message wasn't used.
+     */
+    private RestAction<Message> sendChatGptAttempt(String questionFirstMessage,
```

Comment:
it is not sending anything, might consider a method rename

Reply:
How do you mean? This method sends either the AI generated message or a fallback message to the thread depending on the outcome.